auth/saml/saml_test.go

```diff
@@ -0,0 +1,193 @@
lpSRG5xOFpJQTFVTENiSzdOTXBnYk83eU5sNFRLYkJOQnNBU1dwSGZQSXFmRllZZDQxRjBZQXhmOTE3ZXJJd2l2MGhqNytqSFVMa05ZQlROSk1pVUpJWW5ZN1ZnSmRrQUJ5R1l2T1VadHB6czBlck5lWG9yNVZSU085VUVVVzAyMWVKemF6ZmFtdFZDK2hVRnRnbXd4bkJwZ3dkU3JYdkk2SXFtMFdPU2w2RnJkenRuQm12Vm83LzhyQlRYc2JJV25wT1hxVU9SZmJNTkRpd2F4S2hIS1VKV3U3TmVtUUNMWERjaVM4WEtTMFpMcUZTVGdlUVVldEV0Wm9yR3ovdVg4MlBISjMzZEp5WlNoOWUrVTk0bXZhbDJHbE02b3hvbGpYVzk0ZVc0ZXhuTjNqU1E5enBQTlpselhxdWZvV2dsN2hzMVFjME8xVU1rMHlQRTlpSzdQZlNYMlRiRDYweTJSNU9qQlZmY3FNTERNM0RPdlhDTWwwRFF1cm5jOEtPaTRqSHMzM0MrNkVFeHZnTWJ3ZTNBZXVYaXdSaGkweWZ1Nm1rd1dXM3NZY28wZHEySExBanFwbGRZeTFMM3o5blpNdUxZSFR6bXBSUVFwSC9seVFCQTJGM0tjY21rcU9MTTRKYmorRW9YQjJqL01aOGpZRHNFZGVzaDBRQ3hIWXJJQmFPaEhxelhabTJ6SDZ5UnpuSHpkci9BakNLbzZ1UkNpTW1qb0lmK0VPeVVWcXFoTDg4MHBza21VbUFaYXEvazNjTnZZQXdyQy94UFl3L2k4MFUyYkVtZlZMc2NKMGtqWUhTK21nRGtpc2JIZllwVzdjdWxvbmY4ejFid3JYV1U1RTZNeStzeDNOallNQklTSzhMZWFtUGZHZGJGR1RJZjJRWXZ2ak4yNk5ONTl6R05Ma2hLbVR5bjlpOENsbEJJYlVvSURuTXlBTTdza0RSRHhOU0ttMGQ5YStZOWJ3MmQxeW5YUm80Rm85MmdwdWp0TVp5bzBZcHRrTVVHb3k0RXM5cjFTMEhGWGVWUmI4dHJNYzNqOFdHYzZXM3NUMUdtUEVxUktPNU45KzF4ZDNnN1B1bTVQNWhEOXoxWGlIb3BrL1dqQWFaUmMxUEdPMXVyQzhkMjZBY2Erb2pHeGlsdFlSY0FnUFBYN0tPMDVqUGlZdWtyUlZnbDlZY1AxYkc1SklHZWl5RUxwQmpHMmhYS0poTHh0eitNUnhKbGRadVEzaThpOGdnUWFJUXdrWGc5RlRBZDJIdWtldDJYMVF5WVpXZ1ZOdlBIcG9JeDNhcEJ6TjZJR1Ivb3B0d0VsZ25uaSttY3lBMytVVUdPUU44Rno0eVJwNkZnUTZnWTk1V1ZuemlCMTN0STJFdzhFTTBOSUNiT210RFdIVHBZVC8zcnZkd3hEY3VWODdzQm9GTWpVWnNZWXY2THFveUV1UnY2bmhZbUhPbFF5WExHeEg0MXFLVlRKSVZMOXVObWVMbVRvQjdNM3hweXpzcFkwTEdYTlV3UUpmSWFDNDdiU3V4SVo5RnpiVlFTVGtTcjAvbmJCY2EwVHdISFlJWDhFVndNTisveEF1RnczK3NnTEUyVG1iS0phSXg1azBSaUdkcXd1dGU0VU5Fcm1yVVV6MkphY09halF6dUhFSkZ2VVVBL2d0eEpNODJNS3NIdUZJQ0xhWW5HUnpDYkxDclcrOVNhajBxMUsybUtVdkM4Y2duSW9IQ0Vlc01rRlpKeXpEZnNlSDYyanIzQ2d6cmU3ZTEyc2k3ZmxKa2J1K3NzcTBYWk9rYWhXSTM2ZWVQMHpjaG8rMkN2eFZYVU1xcXU4ejVTL0s2YWh0aUJvYndjN1NzMkVKbkxzb1BmdzBaNHNVWGpVY1V2eEh2L1kvZU5BcWZvcU5BRVFRQUhjZzlaRTd4NUNRb2NUT1A0U0xmQlI5cDZRTnNFZ1BmbXpiN0RVVmRMZTk1MVZHeTlSV05OVzdqOS9yWWlaaWdoTDRWc0FpWlFuUUFhak9GSUNFQVo4MWR2Z3RRd0EzejBTUkZUMXpuYXJONnhrd081akVoN1lWRTI4RVJNNHUrSUs4R2Jmdjdoeis5ZUNqQVdVV0w0WHNZS0U2ZDkzemdXbjkxbXN4V0NqMTdIQkMwVVREcTFlSEtXRkRHc0hhbWoyRTRUVFBIN2MzSjJ3eHAyQnlhVlBFbHBXeXBlQ0tRTmxObXNtNWgrZWFxTnhRRjBBU2Y4cW9qU1Q0QStFbklDM3RQVFlkeGFBd0kxbUJtYUwxZ0tYb0R2dlEyelNGakc2QUpIY2RiWDNxZm5USzlHanJnTFBmWGxwRyt6V2VLck9LSmtpUEt0SEtZaEVaKzQyaUlIZU43MllTQ1hpM0NPVG5pN2tHZFFwWHJ5S240c0xzNWFkTjYza0pjam1mVXFqZW84aXh2SmlReTRuck5nd1kzbHJBZ085SVk0L3ZLMDhmNjNnUXZCdFpGaXJiM0RWUXdUUlZnT0ZzVG9TN2o2aVRvdUtsRG40UGtodFdqdVNYaWNKYm5IUDVUZEZ1ekRMSkNMMElodGM4OExOWXFFdnlLR01NUTJVamRvdHFXTmR5a2h5dXVweDhlWEIyUDh0YzZjTG5NVnZ2N2piT3hNZTRGVnZzVkFsemlreW9qb2FhK3lYTWVPWUZBV0M1YVVOUmJubEo4QWZWWExzYW40bXhUeEJRTmtLaTNJaFR4MWVNZ2RldURJczZzUjRqeHlQakxpYzlEYWJ4aHdRdEZEUmJQZmpuSURXUzlSM0Ztb1A3MWhjODFqQ3JyMVlnbjU3ZWhGa08zY1VpS3Ric0NLK2RLZjNBRFRxbXd5L1hIOW1LbjVJQUhRTTwveGVuYzpDaXBoZXJWYWx1ZT48L3hlbmM6Q2lwaGVyRGF0YT48L3hlbmM6RW5jcnlwdGVkRGF0YT48L3NhbWwyOkVuY3J5cHRlZEFzc2VydGlvbj48L3NhbWwycDpSZXNwb25zZT4="
```

Comment:
Ah, BTW, please keep the comment above the string declaration, just so we can know what's this cryptographic string.